### ISSUE TITLE
Change modulator/carrier levels dynamically, affecting sustained notes

### ISSUE THREAD

**Issue by chrrsy** [jpcima/ADLplug#78]
Amazing plugin, thank you.
I'd love to see it be possible to change the modulator/carrier levels of held down notes fluidly. Currently, if you change the levels of any operator while holding down a note it won't update until you register a new note. This avoids the need to tediously use multiple parts with different envelopes to create timbral variation in a sound.

**Comment by bryc** [comment]
Agreed. The lack of automation for parameters is holding the plugin back. Feedback, Release, Waveform, are other parameters I'd want to automate, which is possible in JuceOPLVSTi.


Edit: This could be considered a duplicate issue of https://github.com/jpcima/ADLplug/issues/61. I added more in depth comments there.

**Comment by orangemono** [comment]
I have to agree. As an old school Adlib/Sound Blaster fan, i've been looking for a while for something like this. I used the early version of JUCE, and have also tried Plogue PortaFM for the OPL2 emulation. I prefer the layout and workflow of ADLplug (and it's got some great flexible options). Unfortunately without assignable params (particularly ADSR) it's just not going to work for me.

**Comment by bertrandguegan** [comment]
I agree too. ADLplug is such an awesome plugin, but I really miss automatable parameters. That would be a major upgrade. Making music with ADLplug would be even more creative and interesting.
